scm/github.go

```diff
@@ -704,53 +705,83 @@ func (s *GithubSCM) CreateIssue(ctx context.Context, opt *CreateIssueOptions) (*
 		Assignees: opt.Assignees,
 	}
 
-	s.logger.Debugf("CreateRepository: creating issue %s on %s Repository", opt.Title, opt.Repository)
+	s.logger.Debugf("Creating issue %s on %s Repository", opt.Title, opt.Repository)
 	issue, _, err := s.client.Issues.Create(ctx, opt.Organization, opt.Repository, newIssue)
 	if err != nil {
 		return nil, ErrFailedSCM{
 			Method:   "CreateIssue",
-			Message:  fmt.Sprintf("failed to create Issue %s, make sure all the details are correct ", opt.Title),
+			Message:  fmt.Sprintf("failed to create issue %s, make sure all the details are correct ", opt.Title),
 			GitError: err,
 		}
 	}
-	s.logger.Debugf("CreateRepository: done creating issue %s", opt.Title)
+	s.logger.Debugf("Done creating issue %s", opt.Title)
 
 	return toIssue(issue), nil
 }
 
-// GetRepoIssue implements the SCM interface
-func (s *GithubSCM) GetRepoIssue(ctx context.Context, issueNumber int, opt *RepositoryOptions) (*Issue, error) {
+// EditRepoIssue implements the SCM interface
+func (s *GithubSCM) EditIssue(ctx context.Context, issueNumber int, opt *CreateIssueOptions) (*Issue, error) {
+	if !opt.valid() {
+		return nil, ErrMissingFields{
+			Method:  "EditIssue",
+			Message: fmt.Sprintf("%+v", opt),
+		}
+	}
+	issueReq := &github.IssueRequest{
+		Title:     &opt.Title,
+		Body:      &opt.Body,
+		State:     &opt.State,
+		Assignee:  opt.Assignee,
+		Assignees: opt.Assignees,
+	}
+
+	s.logger.Debugf("EditIssue: Editing issue %s on %s Repository", issueNumber, opt.Repository)
+	issue, _, err := s.client.Issues.Edit(ctx, opt.Organization, opt.Repository, issueNumber, issueReq)
+	if err != nil {
+		return nil, ErrFailedSCM{
+			Method:   "EditIssue",
+			Message:  fmt.Sprintf("failed to edit issue #%v, on repository: %s, at organization: %s", issueNumber, opt.Repository, opt.Organization),
```

Comment:
Does this make sense?

`"failed to update issue %d on %s/%s", issueNumber, opt.Organization, opt.Repository`

(note the reordering of org and repo)